### ISSUE TITLE
Does the migration still not init git-lfs? How to migrate work boards?

### ISSUE THREAD

**Issue by Naphier** [MicrosoftDocs/vsts-docs#2676]
We were just testing this out on a small repository and all of our image binaries seemed to come over just fine (nothing more than a 5MB).  So these are probably not tracked by LFS, but not a problem for our small project we migrated. It'll be a bigger problem with larger projects. Does the Git-tfs tool help with this (I'm not seeing any info on that in the github repo).  Also worried that git-tfs is no longer being maintained. What's the suggested path when we need LFS?

Also there seems to be no mention anywhere of how to migrate an agile workboard from a tfs project to a git project. It would be really helpful to have that info or a tool that just does it.
Thanks!

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 44afc4fe-66c8-a745-0037-00748a80beb7
* Version Independent ID: 0ea09401-6a08-d745-ff50-7a7b74c41f32
* Content: [Migrate from TFVC to Git - Azure DevOps](https://docs.microsoft.com/en-us/azure/devops/learn/git/migrate-from-tfvc-to-git)
* Content Source: [https://mseng.visualstudio.com/DefaultCollection/TechnicalContent/_git/devops-learn-pr?path=%2fdocs%2fgit%2fmigrate-from-tfvc-to-git.md&version=GBlive&_a=contents](https://mseng.visualstudio.com/DefaultCollection/TechnicalContent/_git/devops-learn-pr?path=%2fdocs%2fgit%2fmigrate-from-tfvc-to-git.md&version=GBlive&_a=contents)
* Product: **devops**
* Git Login: **samgu**
* Microsoft Alias: **routlaw**

**Comment by steved0x** [comment]
@Naphier I'm trying to find out for you, I have got a few queries out.

Does the info in this issue apply or help? (Or is there more than one git-tfs?)

https://github.com/git-tfs/git-tfs/issues/1241

**Comment by KathrynEE** [comment]
@Naphier - If you are just moving your code base from TFVC to Git, there is nothing you need to do to migrate work items. If you are moving your code from TFS to Azure DevOps Services, then you'll want to check out the articles provided here: https://docs.microsoft.com/en-us/azure/devops/articles/migration-overview?view=vsts.

**Comment by Naphier** [comment]
@steved0x That sort of helps, it doesn't make things any easier, though. Migrating TFVC to Git needs to have an option to set up LFS first, really. Even a user prompt that just says "hey we detected a bunch of binaries that might not track well with Git, would you like to track common extensions with Git lfs?" This solution might work out though. We'll have to test to see.

@KathrynEE Are you saying that we can create a git repo under an existing project that's using TFVC? I didn't see that option before, but I am now seeing it. If that's the case then no board migration is needed like you say.

So my only issue would be correctly dealing with LFS.
We've got a couple of weeks before we start migrating so I'll do some research on it too, but sure would be nice to have a tool in Azure DevOps that just handled the large binaries (I know LFS is a bit of a pain).

**Comment by Naphier** [comment]
On further inspection, since we can have the new git repo in the existing Azure Devops project, then I think we're just going to do manual migrations. Most of our repositories are over 1GB anyway, so it seems manual migration is necessary and we can just rely on the TFVC repo's history. We just need to do a freeze and make sure there's nothing we want to revert!

**Comment by Naphier** [comment]
Ya bummer but seems to be the solution. Thanks for the assistance.

**Comment by steved0x** [comment]
@Naphier Thank you again for your feedback, I'm closing this issue.
